girder/api/rest.py

```diff
@@ -170,7 +170,8 @@ def retVal(user, token):
         return retVal(None, token)
     else:
         try:
-            ensureTokenScopes(token, TokenScope.USER_AUTH)
+            ensureTokenScopes(token, getattr(
+                cherrypy.request, 'requiredScopes', TokenScope.USER_AUTH))
```

Comment:
Good point, I'm OK with this if the alternative is an error, I just wanted to be sure the alternative wasn't continuing on without validating scopes at all.